.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,53 @@
+name: Spellcheck
+
+on: [pull_request]
+
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with: 
+          node-version: '14.x'
+
+      - name: Spellcheck
+        uses: lingeorge88/amplify-ghActions-tests
```

Comment:
In the main code base, should this be pointing to the canonical Amplify repo?

Reply:
Hi Glenn, thank you for pointing this out, I meant to test this action on my own forked branch but when I made a pull request to the forked branch it also requested a pull request to the main branch, how could I test this action locally first?

Reply:
If you are trying to merge a branch in your fork to the fork's main branch, you'll have to specify `lingeorge88/main` in this box on the PR page of your forked repo

![forks](https://github.com/ProgramEquity/amplify/assets/54147446/dbb9f475-94f1-483e-b912-878b5378b66f)